src/pages/Booking/components/Theater/ThaterList.jsx

```diff
@@ -0,0 +1,294 @@
+import React, { useContext } from 'react';
+import { TheaterContext } from './Theater';
+import styled from 'styled-components';
+
+const ThaterList = () => {
+  const {
+    theaterCityList,
+    setTheaterCityList,
+    theaterLoaction,
+    setTheaterLoaction,
+    thaterActive,
+    setThaterActive,
+    theaterSelectList,
+    setTheaterSelectList,
+    loactionActive,
+    setLoactionActive,
+  } = useContext(TheaterContext);
+
+  const handleCityClick = () => {
+    setThaterActive(true);
+  };
+  // useEffect(() => {
+  //   fetch(`http://10.58.6.216:8000/bookings?movie-id=${value}`)
+  //     .then(res => res.json())
+  //     .then(movieId => {
+  //       console.log(movieId);
+  //     });
+  // },[]);
+  const handleLocation = data => {
+    const locationId = data.theater_id;
+    const list = [...theaterLoaction];
+    const clickIndex = clickSearchItem(data);
+    data.click = true;
+    list.splice(clickIndex, data);
+    setTheaterLoaction(list);
+
+    const selectList = [...theaterSelectList];
+    const changeIndex = selectList.findIndex(item => item !== -1);
+    selectList.splice(changeIndex, 0, data);
+    selectList.pop();
+    setLoactionActive(true);
+    setTheaterSelectList(selectList);
+    console.group('------location---------');
+    console.log(data);
+    console.groupEnd();
+
+    fetch(`http://10.58.6.216:8000/bookings?theater_id=${locationId}`)
+      .then(res => res.json())
+      .then(value => {
+        console.log(value.TIMETABLE);
+      });
+  };
+
+  const handleDelete = data => {
+    data.click = false;
+    const selectList = [...theaterSelectList];
+    const deleteIndex = deleteSearchItem(data);
+    selectList.splice(deleteIndex, 1);
+    selectList.push(undefined);
+    setTheaterSelectList(selectList);
+  };
+
+  const deleteSearchItem = data => {
+    return theaterSelectList.findIndex(item => item === data);
+  };
+  const clickSearchItem = data => {
+    return theaterLoaction.findIndex(item => item.location === data.location);
+  };
+
+  return (
+    <>
+      <TheaterTab>
+        <TheaterTabBtn role="button">전체</TheaterTabBtn>
+        <TheaterTabBtn role="button" primary>
+          특별관
+        </TheaterTabBtn>
+      </TheaterTab>
+      <TheaterListWrapper>
+        <TheaterCityList>
+          {theaterLoaction.map((data, idx) => {
+            if (idx === 1) {
+              return (
+                <TheaterCityBtn
+                  role="button"
+                  key={idx}
+                  onClick={() => handleCityClick(data)}
+                  className={thaterActive && 'isActive'}
+                >
+                  {data.city}
+                </TheaterCityBtn>
+              );
+            }
+          })}
+          {theaterCityList.map((data, idx) => {
+            if (data.city !== '서울') {
+              return (
+                <TheaterCityBtn
```

Comment:
```suggestion
          {theaterCityList.map((data, idx) => {
            return data.city !== '서울' && <TheaterCityBtn
```
이것도 && 연산자로 사용할 수 있을거 같네요